scene/gui/text_edit.h

```diff
@@ -122,6 +122,7 @@ class TextEdit : public Control {
 	};
 
 private:
+	const int right_side_buffer = 5;
```

Comment:
Better way to register this value, if it should be registered like this:
```suggestion
	static constexpr int RIGHT_SIDE_BUFFER = 5;
```
I'd also suggest making it all caps to distinguish it as a constant
Though alternatively it should just be a constant value where it is used, as it's used just once